assets/src/edit-story/components/panels/sizeAndPosition.js

```diff
@@ -53,16 +80,47 @@ function SizePanel({ selectedElements, onSetProperties }) {
           newHeight = dataPixels(newWidth / ratio);
         }
       }
-      return { width: newWidth, height: newHeight };
+      return { ...state, width: newWidth, height: newHeight };
     });
     evt.preventDefault();
   };
+
+  const handleClick = () => {
+    const newState = { ...state, isFill: !state.isFill };
+    setState(newState);
+    onSetProperties(newState);
+  };
+  const handleSetBackground = () => {
+    const newState = {
+      ...state,
+      isBackground: true,
+      opacity: 100,
+      overlay: null,
+    };
+    setState(newState);
+    const backgroundId = selectedElements[0].id;
+    setBackgroundElement({ elementId: backgroundId });
+    onSetProperties(newState);
+  };
+
+  const flipOptions = [
+    { name: __('None', 'web-stories'), value: null },
+    { name: __('Vertical', 'web-stories'), value: 'vertical' },
+    { name: __('Horizontal', 'web-stories'), value: 'horizontal' },
+    { name: __('Horizontal & Vertical', 'web-stories'), value: 'both' },
```

Comment:
In designs, the flip is controlled with two toggles. So I'm wondering if it'd be also easier to do the same in our data model? E.g. either `{flipHorizontal: boolean, flipVertical: boolean}` or `{flip: {horizontal: boolean, vertical: boolean}}`. Though it's not a very critical change.

Reply:
I was wondering the same actually 👍

Reply:
Yep, definitely. Thought about it and then forgot, thanks for noting this, will change.

Reply:
Should be addressed now.